client/src/screens/Product.js

```diff
@@ -1,14 +1,24 @@
-import React, { useEffect } from 'react'
+import React, { useEffect, useState } from 'react'
 import { useSelector, useDispatch } from 'react-redux'
 import { useRoute } from '@react-navigation/core'
-import { View, FlatList, Image, Text, Button, Divider } from 'native-base'
+import { View, FlatList, Image, Text, Button, Divider, Input } from 'native-base'
 import Loading from '../components/Loading'
 import { fetchProductById } from '../features/product'
+import { putAnswer } from '../utils/api'
 
 export default ({ navigation }) => {
   const route = useRoute()
-  const { product, loading } = useSelector((state) => state.product)
   const dispatch = useDispatch()
+  const [qestionIdx, setQnIdx] = useState()
```

Comment:
Idx is not a common abbreviation for index.
```suggestion
  const [qestionIndex, setQestionIndex] = useState()
```